infrastructure/containers/postgres/automatic-backup/backup_or_restore.py

```diff
@@ -315,23 +315,25 @@ def restore_database(backup_object_name_to_restore_from__override: Optional[str]
     """
     pg_env_vars = {"PGPASSWORD": os.environ["POSTGRES_PASSWORD"]}
 
-    print("Dropping database {db_name}".format(db_name=os.environ["POSTGRES_DB"]))
-    drop_db_cmd = "dropdb --host={host} --port={port} --username={user} {db_name}".format(
-        host=os.environ["POSTGRES_HOST"],
-        port=os.environ["POSTGRES_PORT"],
-        user=os.environ["POSTGRES_USER"],
-        db_name=os.environ["POSTGRES_DB"],
-    )
-    run_shell_command(command=drop_db_cmd, env_vars=pg_env_vars)
-
-    print("Creating empty database {db_name}".format(db_name=os.environ["POSTGRES_DB"]))
-    create_db_cmd = "createdb --host={host} --port={port} --username={user} {db_name}".format(
-        host=os.environ["POSTGRES_HOST"],
-        port=os.environ["POSTGRES_PORT"],
-        user=os.environ["POSTGRES_USER"],
-        db_name=os.environ["POSTGRES_DB"],
-    )
-    run_shell_command(command=create_db_cmd, env_vars=pg_env_vars)
+    # print("Dropping database {db_name}".format(db_name=os.environ["POSTGRES_DB"]))
```

Comment:
Does the new command you added to the job for restoring the database? If so, we can delete all this commented code. We can always go back in time using git if we want to find this again.

Reply:
I was having serious trouble if I had the backup include the drop and create db functions so the backups only fill an emtpy rootski_db and we manually drop and recreate the rootski db. so this is very intentional

Reply:
Oh gotcha. In that case, can you delete this commented code and add some notes to the docstring how this is meant to be used? Like, you can just say that this only works on an empty database where the `POSTGRES_DB` doesn't already exist. If there is data in the database, and we need to restore from the latest backup, we'll need to manually wipe the database volume or tear down/spin up a new lightsail instance from scratch... and THEN run this target.